<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cbControlPanel</class>
 <widget class="QWidget" name="cbControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>42</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CiberRato Control Panel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_Reset">
       <property name="toolTip">
        <string>Reset</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/control/reset</normaloff>:/images/control/reset</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_StartStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Start</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/control/start</normaloff>
         <normalon>:/images/control/stop</normalon>:/images/control/start</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Robot"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Remove">
       <property name="toolTip">
        <string>Kill Robot</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/control/remove</normaloff>:/images/control/remove</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Time">
       <property name="toolTip">
        <string>Toggle Time Limit</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/buttons/time_off</normaloff>
         <normalon>:/images/buttons/time_on</normalon>:/images/buttons/time_off</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Collisions">
       <property name="toolTip">
        <string>Toggle Collisions between Robots</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/buttons/collisions_off</normaloff>
         <normalon>:/images/buttons/collisions_on</normalon>:/images/buttons/collisions_off</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Lock">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Toggle Robot Registrations</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/buttons/lock_closed</normaloff>
         <normalon>:/images/buttons/lock_open</normalon>:/images/buttons/lock_closed</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ManageRobots">
       <property name="toolTip">
        <string>Robot Management</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="default.qrc">
         <normaloff>:/images/buttons/mouse</normaloff>:/images/buttons/mouse</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="default.qrc"/>
 </resources>
 <connections/>
</ui>
